src/extension.ts

```diff
@@ -41,54 +41,41 @@ const formatText = (text: string) => `\r${text.split(/(\r?\n)/g).join("\r")}\r`;
 function handleExit(code: number | null, signal: string | null) {
 	var actionHint: string = '\tPlease report this issue at https://github.com/probe-rs/probe-rs/issues/new';
 	if (code) {
-		vscode.window.showErrorMessage("ERROR: `probe-rs-debugger` exited with an unexpected code: ".concat(`${code}`).concat(actionHint));
+		vscode.window.showErrorMessage("ERROR: `probe-rs-debug` exited with an unexpected code: ".concat(`${code}`).concat(actionHint));
 	} else if (signal) {
-		vscode.window.showErrorMessage("ERROR: `probe-rs-debugger` exited with signal: ".concat(`${signal}`).concat(actionHint));
+		vscode.window.showErrorMessage("ERROR: `probe-rs-debug` exited with signal: ".concat(`${signal}`).concat(actionHint));
 	}
 }
 
 // Messages to be sent to the debug session's console. 
-// Anything sent before or after an active debug session is silently ignored by VSCode. 
-// Any local (generated directly by this extension) messages MUST start with 'ERROR', or 'INFO' , 'WARN', `DEBUG`, or `TRACE` to match the RUST log behaviour. 
-// Any local messages that start with `CONSOLE` will ALWAYS be logged.
-// Any messages that come from the `probe-rs-debugger` STDERR will always be logged, and will already conform with the RUST LOG setting.
+// Any local (generated directly by this extension) messages MUST start with 'ERROR', or 'probe-rs-debug' , or `DEBUG`. 
+// Any messages that start with `ERROR` or `probe-rs-debug` will ALWAYS be logged.
+// Any messages that come from the `probe-rs-debug` STDERR will always be logged.
```

Comment:
Why do we log messages to STDERR and on the direct MS-DAP channel? :)

Reply:
When the extension fires up the `probe-rs-debugger` process (versus connect to an already running process), then it has to monitor and report potential problems that happen outside the context of a DAP session (e.g. process doesn't start, DAP session cannot connect, server dies unexpectedly, Rust panics, etc.). These errors can be either OS related, or Rust ERROR messages that happen before or after the DAP sesion.  The only way to catch them is for the extension to monitor the STDERR from the process it has spawned.